eng/Packages.Data.props

```diff
@@ -120,7 +120,7 @@
     <PackageReference Update="Azure.Messaging.EventHubs" Version="5.12.1" />
     <PackageReference Update="Azure.Messaging.EventGrid" Version="4.21.0" />
     <PackageReference Update="Azure.Messaging.EventGrid.SystemEvents" Version="1.0.0-beta.5" />
```

Comment:
Please rebase on `main` and ensure that only the changes that you're intending to make are in the PR.